common/src/main/scala/org/apache/celeborn/common/RssConf.scala

```diff
@@ -1102,29 +1199,38 @@ object RssConf extends Logging {
       .toSequence
       .createOptional
 
-  val WORKER_FLUSH_BUFFER_SIZE: ConfigEntry[Long] =
-    buildConf("celeborn.worker.flush.buffer.size")
+  val WORKER_FLUSHER_BUFFER_SIZE: ConfigEntry[Long] =
+    buildConf("celeborn.worker.flusher.buffer.size")
       .withAlternative("rss.worker.flush.buffer.size")
       .categories("worker")
       .doc("Size of buffer used by a single flusher.")
       .bytesConf(ByteUnit.BYTE)
       .createWithDefaultString("256k")
-
   def workerHeartbeatTimeoutMs(conf: RssConf): Long = conf.get(WORKER_HEARTBEAT_TIMEOUT)
 
   def workerReplicateThreads(conf: RssConf): Int = conf.get(WORKER_REPLICATE_THREADS)
 
   def workerCommitThreads(conf: RssConf): Int = conf.get(WORKER_COMMIT_THREADS)
 
-  def workerFlushBufferSize(conf: RssConf): Long = conf.get(WORKER_FLUSH_BUFFER_SIZE)
+  def workerFlusherBufferSize(conf: RssConf): Long = conf.get(WORKER_FLUSHER_BUFFER_SIZE)
 
-  def flushTimeout(conf: RssConf): Long = {
-    conf.getTimeAsSeconds("rss.flush.timeout", "120s")
-  }
+  val WORKER_COMMIT_FILES_TIMEOUT: ConfigEntry[Long] =
+    buildConf("celeborn.worker.commit.files.timeout")
```

Comment:
the "files" is quite generic here, if the commit "object" always be shuffle data, how about "celeborn.worker.shuffle.commit.timeout"? if not, we can use 
 "celeborn.worker.commit.timeout" for general, and can add something later like 

```
buildConf("celeborn.worker.A.commit.timeout")
  .fallbackConf(WORKER_COMMIT_TIMEOUT)

buildConf("celeborn.worker.B.commit.timeout")
  .fallbackConf(WORKER_COMMIT_TIMEOUT)
```

Reply:
"celeborn.worker.shuffle.commit.timeout" should be a good choice.  I'll change it.